tests/dummy/app/pods/docs/template.hbs

```diff
@@ -27,6 +27,11 @@
     {{nav.item 'Rows and Trees' 'docs.guides.body.rows-and-trees'}}
     {{nav.item 'Row Selection' 'docs.guides.body.row-selection'}}
     {{nav.item 'Occlusion' 'docs.guides.body.occlusion'}}
+
+    {{nav.section 'Testing'}}
```

Comment:
Is this on the live site? Or `tests/dummy/` is only content for test?

Reply:
This is what generates the "Testing" section in the addon docs. (The `tests/dummy` app is the addon docs.)